pixel-saver@deadalnix.me/buttons.js

```diff
@@ -48,7 +48,11 @@ function createButtons() {
 	let orders = order.replace(/ /g, '').split(':');
 	
 	orders[0] = orders[0].split(',');
-	orders[1] = orders[1].split(',');
+	
+	// Checking the secondary button orders
+	if(typeof orders[1] != 'undefined') orders[1] = orders[1].split(',');
+	// if doesn't exist then use the default orders
+	else orders[1] = orders[0];
```

Comment:
That is not correct. That would duplicate the button on the left and on the right and you'd end up with with some implementation defined mess. Why not something like:

    if (!(1 in orders)) {
        orders[1] = '';
    }